### PR TITLE
IMP email as username and fix defaults

### PR DESCRIPTION
**Pull request by lesykm**

### CODE REVIEW

File: netfilter_openvpn.py [hunk 1]
```diff
@@ -392,7 +418,7 @@ def del_chain(usersrcip, dev):
 	iptables('-D FORWARD -s ' + usersrcip + ' -j ' + usersrcip, False)
 	iptables('-F ' + usersrcip, False)
 	iptables('-X ' + usersrcip, False)
-	ipset("--destroy " + usersrcip, False)
+	ipset("destroy " + usersrcip, False)
```
Comment (c4urself) [inline netfilter_openvpn.py]: let's keep the style they had, single quotes

File: scripts/vpn-fw-find-user.sh [hunk 2]
```diff
@@ -51,6 +51,6 @@ for userip in $useriplist; do
     iptables -L $userip -v -n
     echo
     echo -e "\n--- IPSET HASH TABLE ---"
-    ipset --list $userip
+    ipset list $userip
```
Comment (c4urself) [inline scripts/vpn-fw-find-user.sh]: let's be safe and use `${}`

File: netfilter_openvpn.py [hunk 3]
```diff
@@ -287,10 +304,18 @@ def load_group_rule(usersrcip, usercn, dev, group, networks, uniq_nets):
...
-			# create one rule for each direction
-			build_firewall_rule(usersrcip, usersrcip, destip, '', '', comment)
-			build_firewall_rule(usersrcip, destip, usersrcip, '', '', comment)
+		for destination in fetch_ips_from_file(fd):
+			destarray = destination.split(':')
+			destip = destarray[0]
+			destport = ''
+			if len(destarray) >= 2:
+				destport = destarray[1]
+				for protocol in ['tcp', 'udp']:
```
Comment (c4urself) [inline netfilter_openvpn.py]: use a tuple here

File: netfilter_openvpn.py [hunk 4]
```diff
@@ -78,6 +78,10 @@
 if hasattr(config, 'LDAP_MEMBER_ATTR'):
```
Comment (c4urself) [inline netfilter_openvpn.py:78]: let's reindent this whole file to 4-space tabs, seems like the formatting is a bit strange

Comment (c4urself) [reply netfilter_openvpn.py:78]: will their logging work at all? (the mozdef stuff)

Comment (lesykm) [reply netfilter_openvpn.py:78]: nope, that's why I disabled it in config and remove from dependencies, but I do not want clean it out now

Comment (lesykm) [reply netfilter_openvpn.py:78]: What win will we get from re indenting file(tab -> 4 spaces) besides huge commit?

Comment (c4urself) [reply netfilter_openvpn.py:78]: if we're going to use this I'd like to see logging in place so we have an idea of what's going on; we can skip reindenting for now

Comment (lesykm) [reply netfilter_openvpn.py:78]: `USE_SYSLOG = True` mean we will get all log messages in openvpn.log (already checked)

**Review by c4urself — APPROVED** [review]

**Pull request merged**